src/main/java/frc/robot/subsystems/Motion.java

```diff
@@ -84,6 +84,9 @@ private void setWristPivotAngle(Angle angle) {
   }
 
   public void setAllPosition(MechanismPositionGroup positionGroup) {
+    if (isWristInDanger() == true) {
+      setWristPivotAngle(positionGroup.wristAngle);
+    }
     setLiftPosition(positionGroup.liftHeight);
     setElevatorPivotAngle(positionGroup.pivotAngle);
     setWristPivotAngle(positionGroup.wristAngle);
```

Comment:
if wrist in danger, set to closest not danger value, else set to actual desired value